vb.go

```diff
@@ -0,0 +1,54 @@
+package sgbucket
+
+import (
+	"fmt"
+	"sync"
+)
+
+type VbucketNo uint32
+type VbucketSeq uint64
```

Comment:
I think as it's usage permeates the codebase it involves increasingly less type assertions, because everything expects `VbucketNo` or `VbucketSeq` rather than `uint32` or `uint64`

The huge advantages I see are:

* Able to attach methods to the type and move common code surrounding those types down to those methods and  reduce a lot of duplicated code
* Easy migration to making this a `struct` later that needs to encapsulate additional internal state aside from just the underlying `uint32/64` without affecting caller code